### PR TITLE
Login

### PR DESCRIPTION
**Pull request by danielamormocea**
Related to #4 #36 #69 

Screenshot 2024-08-11 at 19 18 48

Screenshot 2024-08-11 at 19 19 48

### CODE REVIEW

File: .DS_Store
Comment (maettuu) [inline .DS_Store:1]: please don't push mac-specific files - it's in the gitignore already, but make sure that it's not added to VCS

File: server/.DS_Store
Comment (maettuu) [inline server/.DS_Store:1]: please don't push mac-specific files - it's in the gitignore already, but make sure that it's not added to VCS

File: server/app/.DS_Store
Comment (maettuu) [inline server/app/.DS_Store:1]: please don't push mac-specific files - it's in the gitignore already, but make sure that it's not added to VCS

File: .env.example
Comment (jemaie) [inline .env.example:1]: Adjusting the content of this file does neither match the branch nor a commit. Please refrain from doing that. We can adjust this file as soon as we bring the repo to a near end.

File: .DS_Store
Comment (danielamormocea) [reply .DS_Store:1]: yep, it was a mistake.. I'm not sure why it got added as it was in .gitignore 🤷 but I'll remove it

File: .env.example
Comment (danielamormocea) [reply .env.example:1]: This change is useful and I'd prefer having it now. I can make a separate commit / branch just with this if you prefer that.

**Review by DarleneQing — APPROVED** [review]

**Pull request merged**